### DISCUSSION TITLE
SpeedTower.py uses deprecated parameter for M204

### DISCUSSION THREAD

**Discussion by p-v-n** [5axes/Calibration-Shapes#86] [Ideas]
According to https://marlinfw.org/docs/gcode/M204.html, M204 S&lt;accel&gt; has been deprecated. "P&lt;accel&gt;" and/or "T&lt;accel&gt;" should be used.

**Comment by 5axes** [comment]
Thanks a lot for your feedback. But I don't really knows if it's necessary to change anything for the moment. Deprecated doesn't means it's not working , the code S is still valid and some old firmware are not using the new notation.  So I will keep this issue open and waiting the feedback of more user.

**Comment by p-v-n** [comment]
Sounds good, thank you!

**Comment by tastyratz** [comment]
It also seems  P&T actually could create possible accuracy complications with Klipper if you're not careful.

https://github.com/Klipper3d/klipper/issues/2492
